dsl_parser/tests/test_get_capability.py

```diff
@@ -0,0 +1,290 @@
+########
+# Copyright (c) 2018 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+from dsl_parser import functions
+from dsl_parser.tasks import prepare_deployment_plan
+from dsl_parser.tests.abstract_test_parser import AbstractTestParser
+
+
+class TestGetCapability(AbstractTestParser):
```

Comment:
Is it possible to set a default value for an input with a capability?

Reply:
Apparently yes! I'll add a test to validate.